src/panels/config/integrations/ha-integration-header.ts

```diff
@@ -142,6 +142,7 @@ export class HaIntegrationHeader extends LitElement {
     }
     .primary {
       font-size: 16px;
+      margin-right: 64px;
```

Comment:
This is not a good idea. Space is limited and most of the time we don't have badges. Now we will always reserve this space. If we do the badges with `float: right` I think the text wrap should automatically work.

Reply:
adjusted